oci-unit-tests/helper/test_helper.sh

```diff
@@ -15,15 +15,51 @@ load_shunit2() {
 
 debug() {
     if [ -n "${DEBUG_TESTS}" ]; then
-        if [ "${1}" = "-n" ]; then
-            shift
-            echo -n "$@"
-        else
-            echo "$@"
-        fi
+	if [ "${1}" = "-n" ]; then
+	    shift
+	    echo -n "$@"
+	else
+	    echo "$@"
+	fi
```

Comment:
Are these re-indentations intentional?  I don't like the idea of mixing things in the same PR; this should be about bind9.  It's OK if you want to propose a new indentation for this file, but I think another PR should be filed for that.

Reply:
Not intentional, I habitually run a whitespace cleanup routine in my editor and I guess it had its own ideas about how things should be indented...  I've backed these changes out.